### PR TITLE
Add default model for Face Detection demo

### PR DESCRIPTION
**Pull request by Linchenn**

### CODE REVIEW

**Review by fengwuyao — APPROVED** [review]
Looks good to me! Thanks Lin.

File: face-detection/demos/live_video/src/index.js [hunk 1]
```diff
@@ -21,16 +21,15 @@ import '@tensorflow/tfjs-backend-webgpu';
...
-import {Camera} from './camera';
-import {setupDatGui} from './option_panel';
-import {STATE, createDetector} from './shared/params';
-import {setupStats} from './shared/stats_panel';
-import {setBackendAndEnvFlags} from './shared/util';
+import { Camera } from './camera';
+import { setupDatGui } from './option_panel';
+import { STATE, createDetector } from './shared/params';
+import { setupStats } from './shared/stats_panel';
+import { setBackendAndEnvFlags } from './shared/util';
```
Comment (mattsoulanille) [inline face-detection/demos/live_video/src/index.js:32]: Nit: Formatting changed

**Review by mattsoulanille — APPROVED** [review]
LGTM, but there are a lot of formatting changes to this file. Were those intentional?

File: face-detection/demos/live_video/src/index.js (hunk 1 above)
Comment (Linchenn) [reply face-detection/demos/live_video/src/index.js]: Done. Thanks!

**Pull request merged**